src/client/results/mostfrequent/users/UserItemsList.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { toggleMostActiveUser } from './../mostFrequentActions';
+
+class UserItemsList extends Component {
+  componentDidMount() {
+    $('.ui.checkbox').checkbox();
+  }
+
+  componentDidUpdate() {
+    $('.ui.checkbox').checkbox();
+  }
+
+  render() {
+    return (
+      <div style={{ height: '300px', overflowY: 'scroll' }}>
+        <table className="ui very basic table">
+          <thead>
+            <tr>
+              <th className="two wide"></th>
+              <th className="three wide"></th>
+              <th className="two wide"></th>
+              <th className="seven wide"></th>
+              <th className="two wide"></th>
+            </tr>
+          </thead>
+          <tbody>
+            { this.props.users.map((userInfo, id) =>
+              <UserItem key={id} userInfo={userInfo} />) }
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+}
```

Comment:
Do you think it matters that MostUsedWords is built with nested `<div>`s rather than a table? It's inconsistent between the two, but I don't think we need to worry about that at all